### PR TITLE
Display error notification when detected in output

### PR DESCRIPTION
**Pull request by matthewcheok**
#17

screen shot 2016-07-14 at 1 12 08 pm

### CODE REVIEW

File: pkg/nuclide-swift/lib/buildsystem/SwiftPMBuildSystem.js [hunk 1]
```diff
@@ -247,6 +247,11 @@ export class SwiftPMBuildSystem {
           highlightLine(editor, line);
         }
       }
+    } else {
+      atom.notifications.addError('Error', {
+        detail: text.substring(text.indexOf("error:") + 7),
+        dismissable: true,
+      });
```
Comment (modocache) [inline pkg/nuclide-swift/lib/buildsystem/SwiftPMBuildSystem.js:254]: Awesome!! I was wondering how to display errors -- now we know!!

That being said, I'm not sure if we need an error in this case. I wrote this code to display test failures in files. If a test failure appears in a file we don't know about for some reason, it's no big deal.

But anyway, this error reporting will come in handy for other spots -- for example, in the case we invoke `sourcekitten` for autocompletion in `SwiftPMAutocompletionProvider.js`, but `sourcekitten` isn't the path to an executable.

**Comment by modocache** [comment]
Awesome stuff! Merging 😎 

This test failure code probably won't make it into the pull request we send facebook/nuclide, though. But you should add this same style of errors for other parts of the package! 💯

**Comment by matthewcheok** [comment]
It seems any command running through `_showTestFailure` would automatically get this for free?

**Comment by modocache** [comment]
Yeah, and that's good, but I think I might end up removing the test failure code before submitting a pull request. It's only half done, more of a proof of concept. There's a FIXME in there somewhere that states the problem: we display test failures, but we never remove them. Plus, we should be using the Nuclide test runner anyway. See #1 for details.

**Pull request merged**